scout/server/blueprints/cases/controllers.py

```diff
@@ -281,22 +281,16 @@ def sma_case(store, institute_obj, case_obj):
     return data
 
 
-def case(store, institute_obj, case_obj):
+def case(
+    store: MongoAdapter, institute_obj: dict, case_obj: dict, hide_matching: str = "yes"
```

Comment:
```suggestion
    store: MongoAdapter, institute_obj: dict, case_obj: dict, hide_matching: str = "yes"
```
```suggestion
    store: MongoAdapter, institute_obj: dict, case_obj: dict, hide_matching: bool = True
```

Reply:
Yes/no kind of data call for a boolean, no?

Reply:
Well, the trouble is navigating arg conversion; between jinja-browser-wsgi-flask-python booleans are handled slightly differently. We've had that before with "false"/"False"/False/false-representations that suddenly fail in some situations. It is so much easier to pass a string and test for it.